tests/integration/memcached/test_add_servers.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+The agent should report instance metrics when multiple servers are
+added at once
+*/
+
+/*SKIPIF
+<?php require('skipif.inc'); ?>
+*/
+
+/*INI
+*/
+
+/*EXPECT_METRICS_EXIST
+Datastore/instance/Memcached/host1/1
+Datastore/instance/Memcached/host2/2
+Datastore/instance/Memcached/host3/11211
+*/
```

Comment:
This test should not generate errors, correct? If so, please add this expectation:
```suggestion
*/

/*EXPECT_ERROR_EVENTS null */
```

Reply:
As of [e141481](https://github.com/newrelic/newrelic-php-agent/pull/958/commits/e141481475416099da72d1cab770d748844640e8), we no longer expect no errors

Reply:
I still don't see `/*EXPECT_ERROR_EVENTS null */` in this test.

Reply:
With the added test cases, PHP has warnings about argument types begin incorrect, so there are now errors expected.

Reply:
If that's the case, then please separate the test cases that don't generate errors from the ones that do and add correct `/*EXPECT_ERROR_EVENTS` expectations.

Reply:
split in [ffddd54](https://github.com/newrelic/newrelic-php-agent/pull/958/commits/ffddd547e266668dba56be35cffe1a8ce4ea20a7)